syntax = "proto3";

package bindiff;

service BinExportService {
  rpc UploadBinExport (BinExportRequest) returns (BinExportResponse);
}

message BinExportRequest {
  bytes binexport_data = 1;
}

message BinExportResponse {
  repeated string function_names = 1;
}
